src/BlazorUI/Bit.BlazorUI/Styles/bit-css.scss

```
@@ -1,4 +1,4 @@
-@import "theme-variables.scss";
+@import "functions.scss";
```

Stylelint (17.11.1)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)